### PR TITLE
loading page

### PR DESCRIPTION
**Pull request by preigile**
hz what is better

### CODE REVIEW

**Comment by denis1stomin** [comment]
In my browser `Okeears` text is not on the same place as on Editor page

File: webui/src/components/Animation/Spinner.vue [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <template>
     <div class="loading-spinner-default">
-        <div/><div/><div/><div/><div/><div/><div/><div/><div/><div/><div/><div/>
+        <div></div><div></div><div></div><div></div><div></div><div></div>
```
Comment (denis1stomin) [inline webui/src/components/Animation/Spinner.vue:3]: Why do we need duplicate elements?

Comment (preigile) [reply webui/src/components/Animation/Spinner.vue:3]: Because we use it in index.html before loading vue. And this component we use as usual

**Comment by preigile** [comment]
> In my browser Okeears text is not on the same place as on Editor page

Yes, you right! Thanks! I've fixed it

**Review by denis1stomin — APPROVED** [review]
raskoshno =)

**Pull request merged**